bin/gutenberg-packages-update-pr-body.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Script to construct PR body for updating gutenberg packages from list of packages.
+ *
+ * Usage:
+ * node bin/gutenberg-packages-update-pr-body.js [packages-list]
+ *
+ * Example:
+ * node bin/gutenberg-packages-update-pr-body.js "@wordpress/api-fetch@6.8.0 @wordpress/autop@3.11.0"
+ *
+ * Would Output: **Following packages were updated:**<br/><br/>1. [@wordpress/api-fetch@6.8.0](https://www.npmjs.com/package/@wordpress/api-fetch/v/6.8.0)<br>2. [@wordpress/autop@3.11.0](https://www.npmjs.com/package/@wordpress/autop/v/3.11.0)<br>
+ *
+ */
+
+const args = process.argv.slice( 2 );
+const npmURL = 'https://www.npmjs.com/package/';
+let bodyMessage = '**Following packages were updated:**<br/><br/>';
```

Comment:
Why `<br/>` and not `\n`?

Reply:
PR body is expected to be in markdown so using `<br/>`

Reply:
So then wouldn't it be `  \n` for GitHub-flavored Markdown? I mean, `<br>` will work as well. Just seems maybe a but more tidy.

Reply:
Tried it in this comment and no results in the preview. 

---

New \n line

It's rendering `\n` as it is. I think this method of newline doesn't work in GitHub flavoured markdown 🤔

Reply:
Right. I didn't mean a literal `\n` but an actual newline character, so using the escape sequence in a JS string literal to cause a newline to be output. I would have expected this to work:

```suggestion
let bodyMessage = '**Following packages were updated:**\n\n';
```

Reply:
Got it now. Will test the possibility and update this accordingly.